packages/core/styles/global.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0..800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..700;1,300..600&display=swap');
```

Comment:
Doesn't this read as equivalent to 0..800?

Reply:
No, 0/1 is the indicator for italics, so the tuples can be interpreted as `{{ital 0.0},{wght 300.0-700.0}}` and  `{{ital 1.0},{wght 300.0-600.0}}` where ital 1.0 = italicized and ital 0.0 = regular. Open Sans only goes down to 300, and we only use up to 700 for regular and 600 for ital, so I limited it to avoid over-importing

Reply:
The double tuple is also necessary regardless of whether we use the same font range for both italics and non-italics. If we just fix the old url so it has the correct range format (https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800&display=swap) but don't add in the second tuple, it only imports the normal (non-italic) fonts. So even if we want up to 800 for both, it would need to be https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap